deploy/docs/Kube-Prometheus.md

```diff
@@ -1,9 +1,25 @@
 # kube-prometheus Mixin
 
 If you are already using kube-prometheus, you can use the Prometheus installation from there and send metrics to Sumo
-Logic using a mixin to add the correct remote_write configs and add the `cluster` external_label. The mixin is located
-in this repo at `deploy/kubernetes/kube-prometheus-sumo-logic-mixin.libsonnet`. The defaults assume you're deploying
-Sumo Logic collection via Helm and using few customizations. When deploying collection, disable the built-in Prometheus
+Logic using a mixin to add the correct remote_write configs and add the `cluster` external_label.
```

Comment:
@sumo-drosiek this was the mixin we discussed on the call today right? So have you fixed it so it still works or not?

Reply:
So the question is do we want to do it that way or point to the static file in repository as we do it now?

Reply:
We agreed that customer can generate it using `sumologic/kubernetes-tools`